cluster/cluster-stamp.json

```diff
@@ -2104,7 +2104,7 @@
                         "value": "1000m"
                     },
                     "memoryLimit": {
-                        "value": "512Mi"
+                        "value": "1024Mi"
```

Comment:
Will this increase Limits for all containers/pods?

Reply:
Yep. This policy update would apply to all the pods.